webnn.json

```diff
@@ -375,6 +394,15 @@
           {"name": "options", "type": "reduceMean options", "annotation": "const*", "optional": true}
         ]
       },
+      {
+        "name": "resample",
+        "returns": "operand",
+        "args": [
+          {"name": "input", "type": "operand"},
+         {"name": "options", "type": "resample options", "annotation": "const*", "optional": true}
+        ]
+      },
+
```

Comment:
could you please remove this blank line?

Reply:
ok.